blocks/variables.ts

```diff
@@ -75,25 +69,26 @@ const blocks = createBlockDefinitionsFromJsonArray([
     'extensions': ['contextMenu_variableSetterGetter'],
   },
 ]);
-exports.blocks = blocks;
 
+/** Type of a block that has CUSTOM_CONTEXT_MENU_VARIABLE_GETTER_SETTER_MIXIN */
+type CustomContextMenuVariableBlock = Block&CustomContextMenuVariableMixin;
```

Comment:
In `lists.ts` and `logic.ts` I've tended to give these types short names related to the actual types of blocks that `this` will be.

I think it would be fine for this to be `VariableBlock` (`VariableMixin`, `VariableMixinType`) or something similar.